src/v/cluster/archival/tests/BUILD

```diff
@@ -89,3 +118,203 @@ redpanda_cc_gtest(
         "@seastar//:testing",
     ],
 )
+
+redpanda_cc_btest(
+    name = "segment_reupload_test",
+    timeout = "short",
+    srcs = [
+        "segment_reupload_test.cc",
+    ],
+    deps = [
+        ":service_fixture",
+        "//src/v/cloud_storage",
+        "//src/v/cluster",
+        "//src/v/model",
+        "//src/v/storage",
+        "//src/v/storage/tests:disk_log_builder",
+        "//src/v/test_utils:archival",
+        "//src/v/test_utils:seastar_boost",
+        "@boost//:test",
+        "@seastar",
+        "@seastar//:testing",
+    ],
+)
+
+redpanda_cc_btest(
+    name = "ntp_archiver_test",
+    timeout = "moderate",
+    srcs = [
+        "ntp_archiver_test.cc",
+    ],
+    cpu = 1,
+    deps = [
+        ":service_fixture",
+        "//src/v/bytes:iobuf",
+        "//src/v/bytes:iostream",
+        "//src/v/cloud_storage",
+        "//src/v/cloud_storage/tests:common",
+        "//src/v/cloud_storage_clients",
+        "//src/v/cluster",
+        "//src/v/config",
+        "//src/v/model",
+        "//src/v/net",
+        "//src/v/ssx:sformat",
+        "//src/v/storage",
+        "//src/v/storage:resources",
+        "//src/v/storage/tests:disk_log_builder",
+        "//src/v/test_utils:archival",
+        "//src/v/test_utils:fixture",
+        "//src/v/test_utils:scoped_config",
```

Comment:
I think you need the seastar boost test utils dep here. Without bazel will attempt to generate the xml junit output itself by buffering all test output. If you add the dep then boost will write the output file

Reply:
huh, TIL. thx

Reply:
fixed